package.json

```diff
@@ -159,7 +159,7 @@
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged",
-      "pre-push": "npm run lint-warn"
+      "pre-push": "npm run lint-fix && npm run lint-warn"
```

Comment:
Zostawiłem dwa skrypty na przypadek jeśli ktoś będzie chciał je użyć osobno.
Ale nie wiem, czy nie zmienić tego na 1, który będzie od razu poprawiać i sprawdzać warningi.
Bo teraz dwa razy przechodzimy przez wszystkie pliki

Reply:
Raczej bez większego znaczenia. Skoro tak zrobiłeś to niech już tak zostanie. Czasem się przydaje zrobić osobno, miałem sytuację gdzie fix psuł działanie aplikacji, więc dobrze jest móc sprawdzić bez niego.